forge-gui/res/cardsfolder/upcoming/hag_of_dark_duress.txt

```diff
@@ -0,0 +1,8 @@
+Name:Hag of Dark Duress
+ManaCost:2 B
+Types:Creature Hag Warlock
+PT:2/2
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDiscard | TriggerDescription$ At the beginning of your upkeep, each opponent discards a card.
+SVar:TrigDiscard:DB$ Discard | Defined$ Player.Opponent | NumCards$ 1 | Mode$ TgtChoose
```

Comment:
`Defined$ Opponent`

P.S. 1 is default for NumCards on Discard so you can cut that if you like